test/powershell/Modules/Microsoft.PowerShell.Utility/Group-Object.Tests.ps1

```diff
@@ -7,9 +7,9 @@ Describe "Group-Object DRT Unit Tests" -Tags "CI" {
         $results.Count | Should -Be 2
         $results.Name.Count | Should -Be 2
         $results.Group.Count | Should -Be 4
-        $results.Name | Should -Be aA,AA
-        $results.Group | Should -Be aA,aA,AA,AA
-        ,$results | Should -BeOfType "System.Array"
+        $results.Name | Should -Be aA, AA
+        $results.Group | Should -Be aA, aA, AA, AA
+        , $results | Should -BeOfType "System.Array"
```

Comment:
Please move style fixes in another PR. Below too.

Reply:
VS code's autoformat strikes again. It is in a separate commit at least, but if you'd prefer to do a whole separate PR I can manage. :slightly_smiling_face: